src/base/OpenMCCellAverageProblem.C

```diff
@@ -1487,6 +1488,8 @@ OpenMCCellAverageProblem::initializeElementToCellMapping()
 
   // Check that each cell maps to a single phase
   checkCellMappedPhase();
+
+  openmc_set_seed(1);
```

Comment:
@pshriwise can you please explain this line to me? Would this preclude a user wanting to run Cardinal N times, with different starting seeds?

Reply:
Ah, that's a mistake. Nice catch thanks! I think I was trying to rule out something during testing.